client/src/components/Game/Canvas.js

```diff
@@ -245,7 +246,7 @@ class Canvas extends Component {
             {(index !== usernames.length - 1) ? <span>&rarr;</span> : null}</span>
           ))}
         </div>
-
+        {timeLimit !== '' && userIndex === currentPlayerIndex ? <Timer /> : null}
```

Comment:
It's not necessary to use the conditional operator here, I think you can just do this:
`{timeLimit !== '' && userIndex === currentPlayerIndex && 
      <Timer />
}`

Reply:
`''` is probably a falsy value so i think you can even just do `{timeLimit && userIndex === currentPlayerIndex && <Timer /> }`?

Reply:
Possibly, may need to look it up to be sure.

Reply:
falsy confirmed